### PR TITLE
mozc: restore

### PR DESCRIPTION
**Pull request by mvf**
This restores the `mozc` packages removed with #9266.

### CODE REVIEW

**Comment by Cogitri** [comment]
Does this actually build for you? It was dropped because even with the old protobuf it didn't build

**Comment by maxice8** [comment]
It builds, i'm doing some conversions then i'll merge this

**Comment by mvf** [comment]
@maxice8 My latest version has your changes and a link fix for ARM-musl in local builds. Should be all set now!

File: srcpkgs/mozc/template [hunk 1]
```diff
@@ -1,24 +1,119 @@
...
+		vbin out_linux/Release/mozc_emacs_helper
+		vinstall unix/emacs/mozc.el 644 usr/share/emacs/site-list/emacs-mozc/mozc.el
+	}
+}
+
+fcitx-mozc_package() {
+	short_desc="Mozc engine for fcitx - Client of the Mozc input method"
+	depends="${sourcepkg}-${version}_${revision} fcitx>=4.2.9"
+	homepage="https://fcitx-im.org/"
+	license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/mozc/template]: can you use the spdx identifier for this license?

**Pull request merged**